hrjobcontract/js/src/job-contract/controllers/modal/modal-contract-new.controller.js

```diff
@@ -102,6 +104,8 @@ define([
 
       $rootScope.$broadcast('hrjc-loader-show');
       fetchInsurancePlanTypes().then(function () {
+        return fetchProviders();
```

Comment:
it doesn't seem that `fetchProviders` depends on `fetchInsurancePlanTypes`, so we should execute them in parallel

```js
$q.all([
  fetchInsurancePlanTypes(),
  fetchProviders()
])
  .then(function () {
    $rootScope.$broadcast('hrjc-loader-hide');
  });